eras/shelley/impl/src/Cardano/Ledger/Shelley/API/Validation.hs

```diff
@@ -107,14 +100,6 @@ class
     NewEpochState era ->
     Block (BHeaderView (Crypto era)) era ->
     m (EventReturnType ep (Core.EraRule "BBODY" era) (NewEpochState era))
-  default applyBlockOpts ::
-    forall ep m.
-    (EventReturnTypeRep ep, MonadError (BlockTransitionError era) m) =>
-    ApplySTSOpts ep ->
-    Globals ->
-    NewEpochState era ->
-    Block (BHeaderView (Crypto era)) era ->
-    m (EventReturnType ep (Core.EraRule "BBODY" era) (NewEpochState era))
   applyBlockOpts opts globals state blk =
```

Comment:
Yeah, it's just a `default` signature removal, because it is redundant. `DefaultSignatures` are useful when default implementation has some constraints that aren't present in the actual class function signature. In these cases signatures are identical, thus redundant.